lib/apiservers/portlayer/swagger.yml

```diff
@@ -249,6 +249,8 @@ paths:
             $ref: "#/definitions/VolumeResponse"
         '500':
           description: "Error"
+          $schema:
+            $ref: "#/definitions/Error"
         default:
           description: "error"
```

Comment:
I know this wasn't added as part of this PR, but is a total waste.  It won't do what it was originally meant to do (capture generated client errors).  Defining a catch all error without an http status code feels very wrong to me.